multimodal/setup.py

```diff
@@ -49,6 +49,8 @@
     "nlpaug>=1.1.10,<=1.1.10",
     "nltk>=3.4.5,<4.0.0",
     "openmim>0.1.5,<=0.2.1",
+    "pycocotools",
+    "defusedxml",
```

Comment:
Version caps for the two new packages?

Reply:
Added exact version for both packages. Shall I test for the lowest and highest possible version?